cdap-common/src/main/java/co/cask/cdap/common/http/HttpRequests.java

```diff
@@ -91,13 +92,18 @@ public static HttpResponse execute(HttpRequest request, HttpRequestConfig reques
       }
     }
 
-    if (conn instanceof HttpsURLConnection && !requestConfig.isVerifySSLCert()) {
-      // Certificate checks are disabled for HTTPS connection.
-      LOG.trace("Disabling SSL certificate check for {}", request.getURL());
-      try {
-        disableCertCheck((HttpsURLConnection) conn);
-      } catch (Exception e) {
-        LOG.error("Got exception while disabling SSL certificate check for {}", request.getURL());
+    if (conn instanceof HttpsURLConnection) {
+      if (!requestConfig.isVerifySSLCert()) {
```

Comment:
Can you use `HttpsEnabler` instead? It (tries to) unify all settings related to HTTPs usage.

Reply:
Although that class is in `cdap-security`, but I think you can move it to `cdap-common`.

Reply:
It would make these classes rely on io.netty. Can that be in a separate PR?